src/main/resources/standardTestCases/restart.restart_test_without_recover.test

```diff
@@ -1,6 +1,7 @@
 ignore: true
-name: cromwell_restart_without_recover
+name: restart_tes_without_recover
```

Comment:
"tes" -> "test"

Reply:
it was actually meant for `TES` as TaskExecutionSchemas :p